plugins/modules/database/misc/redis_info.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: redis_info
+short_description: Gather information about Redis servers
+description:
+- Gathers information and statistics about Redis servers.
+options:
+  login_host:
+    description:
+    - The host running the database
+    default: localhost
+  login_port:
+    description:
+    - The port to connect to
+    default: 6379
+  login_password:
+    description:
+    - The password used to authenticate with (usually not used)
+notes:
+- Requires the redis-py Python package on the remote host. You can
+  install it with pip (pip install redis) or with a package manager.
+  U(https://github.com/andymccurdy/redis-py)
+seealso:
+- module: redis
+requirements: [ redis ]
+author: "Pavlo Bashynskyi (@levonet)"
+'''
+
+EXAMPLES = r'''
+- name: Get server information
+  redis_info:
```

Comment:
```suggestion
  community.general.redis_info:
```
You need FQCNs (fully qualified collection names) to use this in roles/playbooks (if you don't use the `collection:` keyword).